src/main/resources/db/migration/V20241025_906__COT-906_CIVIL_WA_2_1_base_flag_addition.sql

```diff
@@ -0,0 +1,8 @@
+-- DTSAM-360: insert civil_wa_1_6 base flag into flag_config table
+INSERT INTO flag_config (flag_name, env, service_name, status) VALUES ('civil_wa_2_1', 'local', 'civil', 'false');
+INSERT INTO flag_config (flag_name, env, service_name, status) VALUES ('civil_wa_2_1', 'pr', 'civil', 'false');
```

Comment:
```suggestion
INSERT INTO flag_config (flag_name, env, service_name, status) VALUES ('civil_wa_2_1', 'local', 'civil', 'true');
INSERT INTO flag_config (flag_name, env, service_name, status) VALUES ('civil_wa_2_1', 'pr', 'civil', 'true');
```

for local and pr we enable as they are the test environments